scripts/SANS/sans/state/StateObjects/StateMoveDetectors.py

```diff
@@ -276,7 +276,13 @@ def __init__(self, data_info):
 
     def _set_conversion_value(self, data_info):
         run_number = data_info.sample_scatter_run_number
-        self.conversion_value = 1000. if run_number <= 2217 else 1.
+
+        # If we are going through init before file was selected, lets assume were using
+        # a newer file so in the probable-case the division is the same as Pos 2
+        # When a run number is entered this re-runs anyway overwriting our assumptions
+        # User files after 2217 use Si. units for x but not y
+        if run_number is None or run_number > 2217:
+            self.conversion_value = 1.
```

Comment:
@DavidFair Don't you want to preserve the default of `1000.` if we are on an older file or am I missing something?

Reply:
We should be fine, this is only called from `__init__` (as it's a builder for the actual object) which already pre-sets it to 1000.

Reply:
Long-term I would like to get rid of the builders and just have the beam centre finder itself (as this is the only place this is used) handle it with a simple check during the running

Reply:
Maybe create an issue